backend/lib/endpoints/schema/feedback.js

```diff
@@ -1,19 +1,32 @@
 const S = require("fluent-schema");
 const { strictSchema } = require("./utils");
 
+const feedbackSchema = strictSchema()
+  .prop("age", S.integer().minimum(1).maximum(150))
+  .prop("covidImpact", S.string())
+  .prop("generalFeedback", S.string())
+  .prop("mostValuableFeature", S.string())
+  .prop("rating", S.integer().minimum(1).maximum(5))
+  .prop("userId", S.string())
+  .prop("whatWouldChange", S.string());
+
+const errorSchema = S.object()
+  .prop("statusCode", S.integer())
+  .prop("error", S.string())
+  .prop("message", S.string());
```

Comment:
I have noticed that this one is repeated across all schemas, can we make it global, or something like `strictSchema`

Reply:
@Zaydme I added a function in the utils to reflect this